### PR TITLE
Buisness electronics are now constructable.

### PR DESCRIPTION
**Pull request by Mazianni**
🆑 
Adds design to engineering fab.
🆑

### CODE REVIEW

File: code/modules/research/fabricators/engineering_fabricator.dm [hunk 1]
```diff
@@ -380,6 +380,10 @@
 	name = "Airlock electronics"
 	build_path = /obj/item/weapon/airlock_electronics
 
+/datum/design/item/engifab/electronics/simple/airlockmoduleb
+	name = "business airlock electronics"
+	path = /obj/item/weapon/airlock_electronics/business
```
Comment (Andrew-Fall) [inline code/modules/research/fabricators/engineering_fabricator.dm:385]: code/modules/research/fabricators/engineering_fabricator.dm:385:error: path: undefined var

**Review by Andrew-Fall — CHANGES_REQUESTED** [review]
code/modules/research/fabricators/engineering_fabricator.dm:385:error: path: undefined var

File: code/modules/research/fabricators/engineering_fabricator.dm (hunk 1 above)
Comment (PsyCommando) [reply code/modules/research/fabricators/engineering_fabricator.dm:385]: @Andromeda-K22 just replace 'path' with 'build_path' on line 385

**Comment by michaelchessall** [comment]
business electronics are based on depreciated code. all orgs use regular airlock electronics now.

**Pull request closed without merging**